### ISSUE TITLE
Add Tokens on Config

### ISSUE THREAD

**Issue by JoaoCampos89** [swaponline/MultiCurrencyWallet#2868]
I am adding 3 erc20 tokens along with btc and eth as the default to swap website

What I did so far was adding tokens on mainnet-default.js and build the project and deploy to firebase hosting from build-mainnet folder.

The issues I am having are:

- It's not clear how to add logo, loading, and the svg tokens to the build of webpack?

- What are the fields I can change and edit on mainnet-default.js, there is a full list?

- When clicking on Exchange menu the default is btc-to-usdt, how to change this to btc-to-eth or other default one?

**Comment by JoaoCampos89** [comment]
Additionally, do you have documentation how to add build files to an existing dex react app?

**Comment by shendel** [comment]
example for options you can look in testnet-default https://github.com/swaponline/MultiCurrencyWallet/blob/master/externalConfigs/testnet-default.js

**Comment by JoaoCampos89** [comment]
Yes, I follow that, is that the list of up to date options? Also I want to add logos to webpack build to add them on the widget options, how is the best way to do that?

**Comment by shendel** [comment]
Logo you can change in client/index.html
window.logoUrl
